Nodes/CV7Node.cpp

```diff
@@ -0,0 +1,211 @@
+/****************************************************************************************
+ *
+ * File:
+ * 		CV7Node.cpp
+ *
+ * Purpose:
+ *		The CV7 node provides wind data to the system using the CV7 wind sensor.
+ *
+ * Developer Notes:
+ *
+ *
+ ***************************************************************************************/
+
+#include "CV7Node.h"
+#include "logger/Logger.h"
+#include "dbhandler/DBHandler.h"
+#include "Messages/WindDataMsg.h"
+#include "utility/Utility.h"
+#include <cstring>
+
+// For std::this_thread
+#include <chrono>
+#include <thread>
+
+#include "wiringSerial.h"
+
+
+#define WIND_SENSOR_SLEEP_MS	100
+#define DATA_OUT_OF_RANGE		-2000.
+
+
+CV7Node::CV7Node(MessageBus& msgBus, std::string portName, unsigned int baudRate)
+	:ActiveNode(NodeID::WindSensor, msgBus), m_Initialised(false), m_fd(-1), m_PortName(portName), m_BaudRate(baudRate),
+	 m_MeanWindDir(DATA_OUT_OF_RANGE), m_MeanWindSpeed(DATA_OUT_OF_RANGE), m_MeanWindTemp(DATA_OUT_OF_RANGE)
+{
+
+}
+
+bool CV7Node::init()
+{
```

Comment:
What happens if the success is false while the m_Initialised is true ? The CV7Node::WindSensor start with a failed initialization ?